files/build_templates/init_cfg.json.j2

```diff
@@ -16,5 +16,43 @@
             "{{crm_res}}_high_threshold": "85"{% if not loop.last %},{% endif -%}
 {% endfor %}
         }
+    },
+    "CONTAINER_FEATURE": {
+        "teamd": {
```

Comment:
Since this is a template file, you can eliminate the duplicate code by utilizing templating, like in the CRM config above. Something like:

```
    "CONTAINER_FEATURE": {
{%- for container in ["bgp", "database", "dhcp_relay", "database", "lldp", "pmon", "radv", "sflow",
                      "snmp", "swss", "syncd", "teamd", "telemetry"] %}
        "{{container}}": {
            "auto_restart": "disabled",
            "high_mem_alert": "disabled"
        }{% if not loop.last %},{% endif -%}
{% endfor %}
    }
```

Also, the default states should be "disabled", as in my example above.

Reply:
Great suggestion! Will fix.

Reply:
Please help me review it again.